build-system/pr-check/remote-tests.js

```diff
@@ -44,6 +44,7 @@ function main() {
   startSauceConnect(FILENAME);
 
   if (!isTravisPullRequestBuild()) {
+    timedExecOrDie('gulp dist --fortesting');
     timedExecOrDie('gulp test --unit --nobuild --saucelabs_lite');
```

Comment:
Move `test --unit` above `dist`, since we can save the time spent running `dist` if the unit tests fail.

There's a new risk here, now that remote tests are allowed to fail: If `gulp dist --fortesting` breaks, we'll never catch it. Not sure how to solve this, but let's punt and consult with @erwinmombay.

Reply:
Sounds good.